### PR TITLE
PKX-328 added django silk

### PR DESCRIPTION
**Pull request by mubtadaali**

### CODE REVIEW

**Review by tasawernawaz — DISMISSED** [review]
Just curious about migrations. The rest of it looks good to me. 👍 I believe this package needs migrations, should we add those migrations as part of this PR?

**Comment by mubtadaali** [comment]
> Just curious about migrations. The rest of it looks good to me. 👍 I believe this package needs migrations, should we add those migrations as part of this PR?

Thanks for pointing it out, forgot to add that.

File: cms/envs/devstack.py [hunk 1]
```diff
@@ -79,8 +80,13 @@
 ################################ DEBUG TOOLBAR ################################
 
 INSTALLED_APPS += ['debug_toolbar']
-
 MIDDLEWARE.append('debug_toolbar.middleware.DebugToolbarMiddleware')
+
+################################ DJANGO SILK ################################
+if DEBUG:
```
Comment (emzubair) [inline cms/envs/devstack.py:86]: `DEBUG` flag is set for the Dev server, shouldn't we add `silk` pkg in requirements?

**Review by emzubair — APPROVED** [review]
👍🏽

**Pull request merged**